knative-kubecon/README.md

```diff
@@ -211,7 +208,7 @@ that orchestrates the build.
 
 ```bash
 # Open in your browser (default credentials: admin/admin)
-echo "https://$(minishift ip):8443/console/project/myproject/browse/pods"
+echo "https://$(ocp4 ip):8443/console/project/myproject/browse/pods"
```

Comment:
`$(minishift ip)` runs `minishift ip` as a shell command. There isn't a shell command called `ocp4` so this will fail. We'll have to do some extra work to figure out the right OpenShift browser URL here.

Reply:
@bbrowning. Thanks for the feedback. I'll continue working on this for the next sprint.